### DISCUSSION TITLE
Easy Ship API

### DISCUSSION THREAD

**Discussion by mrkevinhsiao** [abuzuhri/Amazon-SP-API-CSharp#233] [General]
Would you consider integrating Amazon Easy Ship API into this library?

It helps sellers manage and ship Amazon Easy Ship orders.
https://developer-docs.amazon.com/sp-api/docs/easy-ship-api-v2022-03-23-reference

**Comment by abuzuhri** [comment]
yes we can add it to DOLIST, or if you have developer you can help us and contributing this

**Reply by mrkevinhsiao** [reply]
Please kindly add it to your DOLIST. Thank you for your help
